toothpick-runtime/src/main/java/toothpick/ScopeImpl.java

```diff
@@ -324,21 +325,15 @@ private void installModule(Module module) {
       map = mapClassesToUnBoundProviders;
```

Comment:
For this one I understand the concern, we get, we should synchronize, but here is the point : it never never never crashes. Do we get inconsistent data ? I don't think so, but it seems to me the bucketing should fail and crash, but I don't understand why, it passes.